ultralytics/yolo/engine/trainer.py

```diff
@@ -144,6 +144,7 @@ def __init__(self, cfg=DEFAULT_CFG, overrides=None, _callbacks=None):
 
         # Callbacks
         self.callbacks = _callbacks or callbacks.get_default_callbacks()
+
```

Comment:
@jinensetpal maybe this change is not needed? If so, can you please revert it?

Reply:
Reverted, that was edited in error. Thanks for the catch!